### PR TITLE
feat(init): Suppot for app deployment manifest

### PR DESCRIPTION
**Pull request by jgabrielmaia**

### CODE REVIEW

**Review by ffsantos-dev — APPROVED** [review]
Seems good!

File: cmf-cli/resources/template_feed/init/app_deployment_manifest.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+					  label="Logs Folder" internal="/var/log/cmf/" />
+				</deployContainer:Volumes>
+			</deploy:Step>
+
+			<!-- ui -->
+			<deploy:Step xsi:type="deployContainer:containerService" name="ui" label="UI" image="#{ImageRegistry}#/criticalmanufacturing/ui:#{ImageTag}#">
+				<deployContainer:Volumes>
+					<deployContainer:Volume name="$(CLI_PARAM_AppNameMin)_logs_share" optional="true" label="Logs Folder" internal="/var/log/cmf/" />
+				</deployContainer:Volumes>
+			</deploy:Step>
```
Comment (pedrotinoco) [inline cmf-cli/resources/template_feed/init/app_deployment_manifest.xml:274]: Missing Grafana deploy:Step.

File: cmf-cli/resources/template_feed/init/app_deployment_manifest.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  manifestVersion="1.0"
+  version="#{AppVersion}#"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="urn:cmf:dp:xml:ns:main-v1 dp-main-v1.xsd"
+  >
+
+  <install:Description>$(CLI_PARAM_AppDescription)</install:Description>
+	<install:License version="#{TargetVersion}#"/>
+	<install:LicenseAgreements>
+		<!-- This is just an example of license agreement -->
```
Comment (pedrotinoco) [inline cmf-cli/resources/template_feed/init/app_deployment_manifest.xml:23]: Can you format the file a bit better please?

File: cmf-cli/resources/template_feed/init/app_deployment_manifest.xml [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+		<common:LicenseAgreementInternal documentName="MES License Agreement" />
+	</install:LicenseAgreements>
+
+	<install:Dependencies>
+		<install:Dependency id="@criticalmanufacturing\ui" version="#{TargetVersion}#"/>
```
Comment (pedrotinoco) [inline cmf-cli/resources/template_feed/init/app_deployment_manifest.xml]: Missing "@criticalmanufacturing\analytics

**Pull request closed without merging**